src/NServiceBus.Transport.RabbitMQ/DelayedDelivery/DelayInfrastructure.cs

```diff
@@ -78,28 +78,19 @@ public static void TearDown(IModel channel)
 
         public static StartupCheckResult CheckForInvalidSettings(SettingsHolder settings)
         {
-            var routingTopologySupportsDelayedDelivery = settings.GetOrDefault<bool>(SettingsKeys.RoutingTopologySupportsDelayedDelivery);
             var timeoutManagerDisabled = settings.GetOrDefault<bool>(SettingsKeys.DisableTimeoutManager);
             var externalTimeoutManagerAddress = settings.GetOrDefault<string>(coreExternalTimeoutManagerAddressKey) != null;
```

Comment:
I know that this was not changed by this PR. But I don't like the name of this variable. `externalTimeoutManagerAddress` indicates that this holds an address rather than bool value indicating if address was configured.

Reply:
Very true.